pySDC/tests/test_projects/test_pintsime/test_SwitchEstimator.py

```diff
@@ -0,0 +1,462 @@
+import numpy as np
+import pytest
+
+from pySDC.implementations.problem_classes.DiscontinuousTestODE import DiscontinuousTestODE
+from pySDC.projects.DAE.problems.DiscontinuousTestDAE import DiscontinuousTestDAE
```

Comment:
Please do not import these things in a global way, otherwise other tests fail due to an MPI problem (here: fenics). Put these imports into the test functions.